docs/setup.asciidoc

```diff
@@ -189,12 +189,12 @@ logger.debug("Example message!")
 [[correlation]]
 == Elastic APM Log Correlation
 
-`ecs-logging-python` supports automatically collecting https://www.elastic.co/guide/en/ecs/master/ecs-tracing.html[ECS tracing fields]
+`ecs-logging-python` supports automatically collecting  {ecs-ref}/ecs-tracing.html[ECS tracing fields]
 from the https://github.com/elastic/apm-agent-python[Elastic APM Python agent] in order to
-https://www.elastic.co/guide/en/apm/agent/python/current/log-correlation.html[correlate logs to spans, transactions and traces] in Elastic APM.
+{apm-py-ref}/log-correlation.html[correlate logs to spans, transactions and traces] in Elastic APM.
```

Comment:
This one lands on page that says to refer to `logs.hml` instead, maybe we can update the link while we are at it?

Reply:
Good idea!  Fixed in 4616a197ce4f664a9de77095eeaf483eb8d2c077